docs/hazmat/backends/interfaces.rst

```diff
@@ -509,3 +509,12 @@ A specific ``backend`` may provide one or more of these interfaces.
         :param bytes data: DER formatted certificate data.
 
         :returns: An instance of :class:`~cryptography.x509.Certificate`.
+
+    .. method:: load_pem_x509_csr(data)
+
+        .. versionadded:: 0.9
+
+        :param bytes data: PEM formatted certificate request data.
```

Comment:
I discussed this with glyph a bit on IRC and it was his contention that `load_pem_x509_csr` is better than `load_pem_x509_certificate_signing_request` since pem is already an abbreviation. I am leaning towards leaving it as csr here but could be persuaded otherwise. `load_pem_x509_certificate_signing_request` is a pretty long name though.